src/server/game/Entities/Player/Player.cpp

```diff
@@ -12933,20 +12933,7 @@ void Player::RemoveItem(uint8 bag, uint8 slot, bool update, bool swap)
 
                     // remove held enchantments, update expertise
                     if (slot == EQUIPMENT_SLOT_MAINHAND)
-                    {
```

Comment:
This code block is useless, I don't get why random stats should be removed on specific slots for main hand weapon only.
Plus, according to the old code (before this PR), PROP_ENCHANTMENT_SLOT_3 & 4 _(eq :  PROP_ENCHANTMENT_SLOT_0 & 1)_ are **NEVER** used if the item had a Random Suffix _(eq : Random Property)_. So this case shouldn't even exist